package.json

```diff
@@ -70,5 +84,24 @@
     "mkdirp": "^0.5.1",
     "react-app-rewired": "^1.5.0",
     "redux-devtools-extension": "^2.13.2"
+  },
+  "manifest.webapp": {
```

Comment:
is this information needed in both package.json and manifest.webapp? And d2-manifest overwrites manifest.webapp?

Reply:
d2-manifest uses the `package.json` one to generate it at build, so we only need it in `package.json`.

Reply:
Fixed.